roles/scalelite/defaults/main.yml

```diff
@@ -179,7 +183,8 @@ scalelite_api_resources:
     cpus: "{{ ansible_facts['processor_vcpus'] / 2 }}"
     memory: "{{ ansible_facts['memory_mb']['real']['total'] / 2 }}M"
 
-scalelite_hostname: "{{ inventory_hostname }}"
+scalelite_host: "{{ inventory_hostname }}"
```

Comment:
From the variable names it's not entirely clear what the difference is here

Reply:
Previously, we defined both `scalelite_host` and `scalelite_hostname` in infra-schulcloud.
`scalelite_host` was the host defined for the instance.
`scalelite_hostname` was simply defined as `inventory_hostname`.
For our implementation we need the first one, so I switched to that one, becasue it already existed. To prevent more confusion between the two, I removed `scalelite_hostname` in infra-schulcloud.
See:
https://github.com/hpi-schul-cloud/infra-schulcloud/pull/1558/files#diff-0e6e7e47196fb3de34445f0cafbb81b56b4286f2f55009d05369b729c35dbfb4R14
https://github.com/hpi-schul-cloud/infra-schulcloud/pull/1558/files#diff-ad2ebfdb9280d9b3550d2e0209d9957ee51ddf2425fabd4b9c16e9e71a210eddL4